docs/user/adding-glean-to-your-project.md

```diff
@@ -43,37 +43,40 @@ The [android-components release page](https://github.com/mozilla-mobile/android-
 
 #### Integrating with the build system
 
-In order for the Glean SDK to generate an API for your metrics, a Python environment must be accessible at build time.
-This is done automatically by the [`com.jetbrains.python.envs`](https://github.com/JetBrains/gradle-python-envs/) plugin.
-The plugin **must** be manually enabled by adding the following `plugins` block at the top of the `build.gradle` file for your app module.
+In order for the Glean SDK to generate an API for your metrics, two Gradle plugins must be included in your build:
 
-```Groovy
-plugins {
-    id "com.jetbrains.python.envs" version "0.0.26"
-}
-```
+- The [Glean Gradle plugin](https://github.com/mozilla/glean/tree/master/gradle-plugin/)
+- JetBrains' [Python envs plugin](https://github.com/JetBrains/gradle-python-envs/)
 
-Right before the end of the same file, the Glean SDK build script must be included.
-This script can be referenced directly from the GitHub repo, as shown below:
+The Glean Gradle plugin is distributed on Maven, so we need to tell your build where to look for it by adding the following to the top of your `build.gradle`:
 
-```Groovy
-apply from: 'https://raw.githubusercontent.com/mozilla-mobile/android-components/v{latest-version}/components/service/glean/scripts/sdk_generator.gradle'
 ```
+buildscript {
+    repositories {
+        mavenLocal()
 
-> **Important:** the `{latest-version}` placeholder in the above link should be replaced with the version number of the Glean SDK used by the project.
-For example, if version *6.0.2* is used, then the include directive becomes:
+        dependencies {
+            classpath "org.mozilla.telemetry:glean-gradle-plugin:{latest-version}"
```

Comment:
Is `latest-version` a placeholder here? If so, I'd retain the note we had here for stating that :)

Reply:
Indeed.  I didn't intend to remove this.